connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -531,8 +547,20 @@ private Converter badConverter() {
         return converter;
     }
 
+    private void mockSourceTransform() {
+        FaultyPassthrough<SourceRecord> faultyPassthrough = new FaultyPassthrough<>();
+        @SuppressWarnings("unchecked")
+        Class<? extends Transformation<?>> value = (Class<? extends Transformation<?>>) (Class<?>) FaultyPassthrough.class;
+        OngoingStubbing<Class<? extends Transformation<?>>> transformClass = when(transformation.transformClass());
+        transformClass.thenReturn(value);
```

Comment:
```suggestion
        doReturn(FaultyPassthrough.class).when(transformation).transformClass();
```
Same suggestion as above